spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -45,14 +45,26 @@ interface FungibleTokenPacketData {
   receiver: string
   memo: string
 }
-```
 
+interface FungibleTokenPacketDataV2 {
+  tokens: []Token
+  sender: string
+  receiver: string
+  memo: string
+}
+
+interface Token {
+  denomination: string
```

Comment:
is it an intentional decision to change this from `denom` (V1) to `denomination`?